src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -2796,13 +2796,6 @@ public SectionComponent tSection2Section(Section cdaSection) {
 			fhirSec.setCode(dtt.tCD2CodeableConcept(cdaSection.getCode()));
 		}
 
-		// text -> text
-		if(cdaSection.getText() != null) {
-			Narrative fhirText = dtt.tStrucDocText2Narrative(cdaSection.getText());
-			if(fhirText != null)
-				fhirSec.setText(fhirText);
-		}
-
```

Comment:
is this backout needed for it to work?  Just because we don't want a narrative text doesn't mean someone else won't.

Reply:
We could check the config for generateNarrative if you'd like.

Reply:
changed it to use the config boolean.